src/Raven.Server/Dashboard/DatabasesInfoRetriever.cs

```diff
@@ -142,10 +152,53 @@ public static IEnumerable<AbstractDashboardNotification> FetchDatabasesInfo(Serv
+                        long extRepCountOnNode = GetExtRepCountOnNode(database, dbRecord, serverStore);
+                        
+                        long replicationHubCountOnNode = 0;
+                        var replicationHubCount = database.ReplicationLoader.OutgoingHandlers.Count(x => x.IsPullReplicationAsHub);
+                        replicationHubCountOnNode += replicationHubCount;
+
+                        var replicationSinkCount = dbRecord.SinkPullReplications.Count;
+                        long replicationSinkCountOnNode = GetReplicationSinkCountOnNode(database, dbRecord, serverStore);
+                        
+                        var ravenEtlCount = database.EtlLoader.RavenDestinations.Count;
+                        long ravenEtlCountOnNode = GetRavenEtlCountOnNode(database, dbRecord, serverStore);
+                      
+                        var sqlEtlCount = database.EtlLoader.SqlDestinations.Count;
+                        long sqlEtlCountOnNode = GetSqlEtlCountOnNode(database, dbRecord, serverStore);
+                          
+                        var olapEtlCount = database.EtlLoader.OlapDestinations.Count;
+                        long olapEtlCountOnNode = GetOlapEtlCountOnNode(database, dbRecord, serverStore);
+                        
+                        var periodicBackupCount = database.PeriodicBackupRunner.PeriodicBackups.Count;
+                        long periodicBackupCountOnNode = GetBackupCountOnNode(database, dbRecord, serverStore);
+                            
+                        var subscriptionCount = database.SubscriptionStorage.GetAllSubscriptionsCount();
+                        long subscriptionCountOnNode = GetSubscriptionCountOnNode(database, dbRecord, serverStore, context);
+                       
+                        var ongoingTasksItem = new DatabaseOngoingTasksInfoItem()
+                        {
+                            Database = database.Name,
+                            ExternalReplicationCount = extRepCountOnNode,
+                            ReplicationHubCount = replicationHubCountOnNode,
+                            ReplicationSinkCount = replicationSinkCountOnNode,
+                            RavenEtlCount = ravenEtlCountOnNode,
+                            SqlEtlCount = sqlEtlCountOnNode,
+                            OlapEtlCount = olapEtlCountOnNode,
+                            PeriodicBackupCount = periodicBackupCountOnNode,
+                            SubscriptionCount = subscriptionCountOnNode
+                        };
+                        databasesOngoingTasksInfo.Items.Add(ongoingTasksItem);
+                        
+                        var ongoingTasksCount = extRepCount + replicationHubCount + replicationSinkCount + 
+                                                ravenEtlCount + sqlEtlCount + olapEtlCount + periodicBackupCount + subscriptionCount;
```

Comment:
Can we move the code added above to a single dedicated method that will return `DatabaseOngoingTasksInfoItem` and probably also total `ongoingTasksCount` via `out` parameter?

Reply:
done